back/lib/participation_method/native_survey.rb

```diff
@@ -41,13 +41,7 @@ def default_fields(custom_form)
 
       multiloc_service = MultilocService.new
       [
-        CustomField.new(
-          id: SecureRandom.uuid,
-          key: 'page1',
-          resource: custom_form,
-          input_type: 'page',
-          page_layout: 'default'
-        ),
+        start_page_field(custom_form),
```

Comment:
Now shared between methods. Suspect there may be some overlap here with Edwin's work